languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -10083,6 +10083,52 @@ though, and has since been slightly modified)-->
     </rulegroup>
   </category>
   <category id="CAT_GRAMMAIRE" name="Grammaire">
+      <rulegroup id="VERBE_INFINITIF_OU_PPA" name="verbe infinitif ou ppa" default="temp_off">
```

Comment:
The ID promises more than the rule delivers. We should keep the ID precise enough, so we don't think this is the rule that takes care of any infinitive/ppa problem. We could call it AUX_PPA_INF, for example